WordPress/Sniffs/Theme/FileIncludeSniff.php

```diff
@@ -0,0 +1,68 @@
+<?php
+/**
+ * WordPress Coding Standard.
+ *
+ * @package WPCS\WordPressCodingStandards
+ * @link    https://github.com/WordPress-Coding-Standards/WordPress-Coding-Standards
+ * @license https://opensource.org/licenses/MIT MIT
+ */
+
+/**
+ * Check if a theme uses include(_once) or require(_once) when get_template_part() should be used.
+ *
+ * @link    https://make.wordpress.org/themes/handbook/review/required/#core-functionality-and-features
+ *
+ * @package WPCS\WordPressCodingStandards
+ *
+ * @since   0.xx.0
+ */
+class WordPress_Sniffs_Theme_FileIncludeSniff implements PHP_CodeSniffer_Sniff {
+
+	/**
+	 * A list of files to skip.
+	 *
+	 * @var array
+	 */
+	public $file_whitelist = array(
+		'functions.php' => true,
+	);
+
+	/**
+	 * A list of tokenizers this sniff supports.
+	 *
+	 * @var array
+	 */
+	public $supportedTokenizers = array(
```

Comment:
No need to change this, but as a general remark - I believe the default in PHPCS is `PHP` and adding the property is only needed when you want to sniff JS/CSS or a combination of these, though I'd have to double check the default to be sure.
(If it is the default, then there is no need for the property)